packages/p2p/scripts/extract-string.js

```diff
@@ -5,14 +5,19 @@ const getRegexPattern = () => /(i18n_default_text={?|localize\()\s*(['"])\s*(.*?
 const getStringsFromInput = (input, i18n_marker = getRegexPattern()) => {
     const messages = [];
 
-    while ((result = i18n_marker.exec(input)) !== null) {
-        const extracted = result[3];
-        // Replace escape characters.
-        messages.push(extracted.replace(/\\/g, ''));
+    let continue_loop = true;
+    while (continue_loop) {
+        const result = i18n_marker.exec(input);
+        if (result == null) continue_loop = false;
```

Comment:
```suggestion
    while (true) {
        const result = i18n_marker.exec(input);
        if (result == null) break;
```

Reply:
@balakrishna-binary actually first I did it that way but since Eslint [no-constant-condition](https://eslint.org/docs/latest/rules/no-constant-condition) rule disallows using
while(true) and then just break the loop if result==null , I had to write it like that.